schemas/context/person.schema.json

```diff
@@ -56,6 +56,27 @@
                     },
                     "description": "Gender identity of the person.\n",
                     "default": "not_specified"
+                },
+                "xdm:maritalStatus": {
+                    "title": "Marital Status",
+                    "type": "string",
+                    "enum": [
+                        "married",
+                        "single",
+                        "not_specified"
+                    ],
+                    "meta:enum": {
+                        "married": "Married",
+                        "single": "Single",
+                        "not_specified": "Not Specified"
+                    },
+                    "description": "Marital status of the person.\n",
+                    "default": "not_specified"
+                },
+                "xdm:nationality": {
+                    "title": "Nationality",
+                    "type": "string",
```

Comment:
I would add a pattern match here so that invalid values aren't entered.

Reply:
I'll add this.  Given other fields in XDM use a 2 code pattern should we enforce consistency across the ISO 3166-1 standards to use only 2 or 3 code patterns?

Reply:
Actually, whether to use 2 or 3 may differ based on the use case.  We also use BCP-47 codes in some cases as well

Reply:
Resolved in latest commit

Reply:
@eknee We need to maintain the consistency for the pattern "Countrycode" and "Nationality" as both semantically lean towards each other. Since we already have 3166-1 standards maintained in xdm via CountryCode, it is advisable to have nationality follow that standard.

Reply:
@khanna-vikas this has been updated to use the 3 digit ISO code.